packages/@webex/webex-core/src/lib/services/service-catalog.js

```diff
@@ -247,12 +246,21 @@ const ServiceCatalog = AmpState.extend({
     ];
 
     return serviceUrls.find((serviceUrl) => {
-      if (incomingUrlObj.hostname === Url.parse(serviceUrl.defaultUrl).hostname) {
+      // Check to see if the URL we are checking starts with the default URL
+      if (url.startsWith(serviceUrl.defaultUrl)) {
```

Comment:
question: is it ever at all possible for defaultUrl to be empty? startsWith() would always return true for it

Reply:
while in theory it might be possible for the catalog to have a blank serviceLink (this would be an error in the catalog), the parsing in _formatReceivedHostmap in services.js would ignore that entry as we do the following during parsing:
```js
      try {
        host = new URL(serviceUrl).host;
      } catch (e) {
        return;
      }
```
the defaultUrl is the serviceUrl here so we exclude invalid serviceUrls early on.